src/ctapipe/io/interpolation.py

```diff
@@ -11,6 +11,8 @@
 
 from .astropy_helpers import read_table
 
+dimensionless_unit = u.Unit()
```

Comment:
There e is no need to define this.

There is `u.one` as short  alias or `u.dimensionless_unscaled` if you want to be explicit.